cvmfs/catalog_mgr_rw.h

```diff
@@ -111,6 +123,8 @@ class WritableCatalogManager : public SimpleCatalogManager {
 
   manifest::Manifest *Commit(const bool     stop_for_tweaks,
                              const uint64_t manual_revision);
+  void Balance();
+  void FixWeight(WritableCatalog *catalog);
```

Comment:
Then better private by default. It can still be easily promoted to a public method if we need it at a later point.